importexport/musicxml/importmxmlpass2.cpp

```diff
@@ -737,6 +737,7 @@ static void addLyric(MxmlLogger* logger, const QXmlStreamReader* const xmlreader
             }
       else {
             l->setNo(lyricNo);
+            l->setEven(lyricNo % 2);
```

Comment:
Really? The % operator?

Reply:
I would prefer to do the following `l->setEven(l->isEven())` but doesn't that look even more confusing? Either way, hopefully, @Jojo-Schmitz idea of setting `_even` inside `setNo` will work flawlessly. That will alleviate the need for `setEven`.

Reply:
Why not use  `l->setEven(isEven(lyricsNo))`?

Reply:
I believe that setting `_even` inside `setNo()` is a very bad idea. Unless you mean you are going to compare it against its previous value and call `initTid()` and set `styleDidChange` to `true` if the values differed. Probably best to just let this all happen in one place.

Reply:
@mattmcclinch yes, that's what I was thinking to do. `styleDidChange` is also out of place (since it's usually, if not always) not in *::layout. Frankly, I don't love either option. I prefer setters than only set, but at the same time having to call a second function when setting the number of the lyric is asking for trouble.

Reply:
Sorry, what are the two options that you don't love?

Reply:
The first one is setting `_even` in `setNo`. The second one is having to call 2 functions `setNo` and `setEven` together.

Reply:
Then let's not do either. Leave it the way it is and just change `% 2` to `& 1`.